bin/createsrcdirandfiles.js

```diff
@@ -74,37 +77,39 @@ const createSrcDirAndFiles = (details) => {
                 const sequelizeSetupFile = `${appBaseDirectory}/src/models/setup.js`;
                 const sequelizeSetupFileName = fs.createWriteStream(sequelizeSetupFile);
                 const pathName = sequelizeSetupFileName.path;
-                let modelData;
                 if (orm && orm.toLowerCase() === 'sequelize') {
-                    modelData = dataInFiles.sequelizeSetupData;
+                    /**
+                    * set up data in sequelize here
+                    */
```

Comment:
Will only work if ORM is chosen is `sequelize`, but what about an instance where `no ORM` is chosen even when SQLite is selected. 
Need to check the database before ORM instead.